go/types/rolling_value_hasher.go

```diff
@@ -84,6 +85,9 @@ func (rv *rollingValueHasher) HashByte(b byte) bool {
 	if !rv.crossedBoundary {
 		rv.bz.HashByte(b ^ rv.salt)
 		rv.crossedBoundary = (rv.bz.Sum32()&rv.pattern == rv.pattern)
+		if rv.bw.offset > maxChunkSize {
```

Comment:
Seems unfortunate that we need to do this (if it's just to solve that problem). Worth coming back to I think?

Reply:
yup. added a todo to remove